pkgs/development/libraries/nss/default.nix

```diff
@@ -140,9 +146,12 @@ in stdenv.mkDerivation rec {
        libfile="$out/lib/lib$libname.so"
        LD_LIBRARY_PATH=$out/lib:${nspr.out}/lib \
      '') + ''
-        ${nss}/bin/shlibsign -v -i "$libfile"
+        # NOTE: this makes the build not reproducible.
+        # ${nss}/bin/shlibsign -v -i "$libfile"
```

Comment:
Don't have strong feelings about it, but it's unusual to leave code commented-out like this.  This is expected, yes? :) Just checking.

Reply:
Yes, that was a mistake. Thanks for double-checking :)